core/trino-main/src/main/java/io/trino/sql/gen/CursorProcessorCompiler.java

```diff
@@ -305,9 +306,20 @@ public BytecodeNode visitInputReference(InputReferenceExpression node, Scope sco
 
                 Class<?> javaType = type.getJavaType();
 
+                String recordCursorMethodName;
+                if (javaType == boolean.class || javaType == long.class || javaType == double.class) {
+                    recordCursorMethodName = "get" + Primitives.wrap(javaType).getSimpleName();
+                }
+                else if (javaType == Slice.class) {
+                    recordCursorMethodName = "getSlice";
+                }
+                else {
+                    recordCursorMethodName = "getObject";
+                }
```

Comment:
that's overkill if it is just for description purposes.

Reply:
Thanks for your comment !

> that's overkill 

Agree with this, but I don’t have a better idea to solve this problem... Any suggestions ?

> if it is just for description purposes.

Yes this is merely for fixing description.

Reply:
what about
```
format("cursor.get<%s>(%d)", type, field)
```
?

Reply:
You mean no need to fix this ? If you think doing didn't bring more value, I can close this PR.